Dockerfile

```diff
@@ -1,13 +1,17 @@
-FROM handcraftedbits/nginx-unit-webhook:2.6.5
+FROM handcraftedbits/nginx-unit-webhook:latest
```

Comment:
Can you revert this (I like to use specific version numbers for my images)?  Then I call accept the PR.  Thanks!

Reply:
Sure! I'll do it now!